packages/next/server/config.ts

```diff
@@ -605,39 +612,30 @@ function assignDefaults(dir: string, userConfig: { [key: string]: any }) {
     result.output = 'standalone'
   }
 
-  if (
-    result.experimental &&
-    'transpilePackages' in (result.experimental as any)
-  ) {
-    Log.warn(
-      `\`transpilePackages\` has been moved out of \`experimental\`. Please update your ${configFileName} file accordingly.`
-    )
-    result.transpilePackages = (result.experimental as any).transpilePackages
-  }
-
-  if (
-    result.experimental &&
-    'skipMiddlewareUrlNormalize' in (result.experimental as any)
-  ) {
-    Log.warn(
-      `\`skipMiddlewareUrlNormalize\` has been moved out of \`experimental\`. Please update your ${configFileName} file accordingly.`
-    )
-    result.skipMiddlewareUrlNormalize = (
-      result.experimental as any
-    ).skipMiddlewareUrlNormalize
-  }
-
-  if (
-    result.experimental &&
-    'skipTrailingSlashRedirect' in (result.experimental as any)
-  ) {
-    Log.warn(
-      `\`skipTrailingSlashRedirect\` has been moved out of \`experimental\`. Please update your ${configFileName} file accordingly.`
-    )
-    result.skipTrailingSlashRedirect = (
-      result.experimental as any
-    ).skipTrailingSlashRedirect
-  }
+  warnOptionHasBeenMovedOutOfExperimental(
+    result,
+    'transpilePackages',
+    'transpilePackages',
+    configFileName
+  )
+  warnOptionHasBeenMovedOutOfExperimental(
+    result,
+    'allowMiddlewareResponseBody',
+    'allowMiddlewareResponseBody',
```

Comment:
I'm not seeing an option for `allowMiddlewareResponseBody`

Reply:
Oh, my bad.
`allowMiddlewareResponseBody` was removed by @timneutkens in https://github.com/vercel/next.js/pull/44224. I must have missed it when rebasing my PR.

Fixed in [`7e9482f` (#44213)](https://github.com/vercel/next.js/pull/44213/commits/7e9482fa24c8e18be767bb6b03071ce51d1a4c60).